packages/dapp/components/layout/anvil/anvil-rpcs.tsx

```diff
@@ -49,7 +49,7 @@ export default function AnvilRpcs() {
 
       console.log("result", result);
     } catch (error) {
-      console.log("error", error);
+      /* empty */
```

Comment:
replaced with /* empty */ for eslint.

Reply:
fixed the failing methods being incorrectly typed. Any user error will be logged by rpc result.